src/libraries/Directory.Build.props

```diff
@@ -293,8 +293,7 @@
   <PropertyGroup>
     <EnableDefaultItems>false</EnableDefaultItems>
 
-    <!-- Clear the init locals flag on all src projects, except those in VB, where we can't use spans. -->
-    <ILLinkClearInitLocals Condition="'$(IsSourceProject)' == 'true' and '$(Language)' != 'VB'">true</ILLinkClearInitLocals>
+    <SkipLocalsInit Condition="'$(IsSourceProject)' == 'true' and '$(Language)' == 'C#'">true</SkipLocalsInit>
```

Comment:
Given the conversation at https://github.com/dotnet/runtime/pull/33868#issuecomment-601789626, I don't think the `$(Language)` property works in the `Directory.Build.props` file. Instead, you can use:

`'$(MSBuildProjectExtension)' != '.vbproj'`
or
`'$(MSBuildProjectExtension)' == '.csproj'`

to test for "Is this a C# or VB project?".